modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/resource/releases/SW360Release.java

```diff
@@ -129,19 +130,34 @@ public void setDownloadurl(String downloadurl) {
         this.downloadurl = downloadurl;
     }
 
-
     @JsonIgnore
-    public Map<String, String> getCoordinates() {
+    public Set<String> getPurls() {
         return externalIds.entrySet().stream()
-                .filter(e -> SW360CoordinateKeysToArtifactCoordinates.getValues().contains(e.getKey()))
-                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+                .filter(e -> e.getKey().startsWith(PURL_PREFIX))
+                .map(Map.Entry::getValue)
+                .collect(Collectors.toSet());
     }
 
-    public SW360Release setCoordinates(Map<String, String> coordinates) {
-        externalIds.putAll(coordinates);
+    public SW360Release setPurls(Set<String> purls) {
+        dropAllPurls();
+
+        int i = 1;
+        for (String purl : purls) {
+            if (purl != null && !purl.isEmpty()) {
+                externalIds.put(PURL_PREFIX + i, purl);
```

Comment:
why are the keys just the purls with enumeration, we have a better option. In my opinion it would be better to use the old prefix, something like:
```
   'mvn': 'pkg:maven/org.apache.xmlgraphics/batik-anim@1.9.1'
```
This also allows for better searches via the SW360 rest interface.

Or maybe just use the scheme as key. (yes, there can be multiple purls of the same scheme for the same artifact. But I do not think that this is a problem here)

Reply:
@blaumeiser-at-bosch what do you think?

Reply:
Interesting point, disadvantage here is that I have to know which kind of component that is so searching is easier on the one hand side, because I do not have to search until the index ends, but it is also harder, because I have to find out which types of purls are there.

In general, we should use the same type as key, not anything else, so in the example case, "maven" instead of "mvn"

With scheme you mean the type, right?

Reply:
Yes, I probably mean `type`.